packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/billing/depositMovement/deposit-movement.controller.js

```diff
@@ -74,6 +75,50 @@ export default /* @ngInject */ function TelecomTelephonyBillingAccountBillingDep
       });
   };
 
+  function getCanTransferSecurityDeposit(targetsChunk) {
+    return $q.all(
+      reduce(
+        targetsChunk,
+        (obj, value) => {
+          const billingAccount = get(value, 'value.billingAccount');
+          // eslint-disable-next-line no-param-reassign
+          obj[billingAccount] = OvhApiTelephony.v6()
+            .canTransferSecurityDeposit(
+              {
+                billingAccount: self.source.billingAccount,
+              },
+              {
+                billingAccountDestination: billingAccount,
+              },
+            )
+            .$promise.then((data) => data.value)
+            .catch((err) => {
+              if (err.status === 400) {
+                // means that deposit cannot be transfered
+                return false;
+              }
+              return $q.reject(err);
+            });
+          return obj;
+        },
+        {},
+      ),
+    );
+  }
+
+  async function fetchDataInChunks(targetsArr) {
+    const targetsArrCopy = cloneDeep(targetsArr);
+    let result = {};
+    while (targetsArrCopy.length) {
+      // eslint-disable-next-line no-await-in-loop
+      const resultOfChunk = await getCanTransferSecurityDeposit(
+        targetsArrCopy.splice(0, 10),
+      );
+      result = Object.assign(result, resultOfChunk);
+    }
+    return result;
+  }
```

Comment:
If duable, it's better to complete this task in routing file.
It will avoid us the `async await` by using resolve behavior.